tests/integration/api_v3_product_write.t

```diff
@@ -662,6 +662,21 @@ my $tests_ref = [
 			}
 		}',
 	},
+	# remove to categories (existing)
+	{
+		test_case => 'patch-tags-fields-remove',
+		method => 'PATCH',
+		path => '/api/v3/product/1234567890100',
+		body => '{
+			"fields" : "updated",
+			"product": { 
+				"categories_tags_remove": ["en:tea"],
+				"stores_tags_remove": ["Carrefour", "Mon Ptit magasin"],
+				"countries_tags_fr_remove": ["Italie", "en:spain"],
+				"labels_tags_remove": ["végétarien", "Something unrecognized in French"]
+			}
+		}',
+	},
```

Comment:
also you should have a json file resulting from this test. I let you see the documentation https://openfoodfacts.github.io/openfoodfacts-server/dev/how-to-use-gitpod/

Reply:
I have made a few changes to the code and want to test it, but I'm having trouble getting a json file with the expected results. I'm running this command: `make test-unit="api_v3_product_write.t --update-expected-results"`

Reply:
**
![Uploading Screenshot 2024-03-25 at 12.53.35.png…]()
**

Reply:
Yes I can run the website locally